pages/k8s/keepalived.md

```diff
@@ -0,0 +1,85 @@
+---
+wrapper_template: "base-docs.html"
+markdown_includes:
+  nav: "shared/_side-navigation.md"
+context:
+  title: "HA for kubeapi-load-balancer"
+  description: Using keepalived to provide failover for the kube-api-loadbalancer .
+keywords: failover, ha, kube-api-loadbalancer, loadbalancer, keepalived
+tags: [operating]
+sidebar: k8smain-sidebar
+permalink: keepalived.html
+layout: [base, ubuntu-com]
+toc: False
+---
+
+The standard deployment of  the
+**Canonical Distribution of Kubernetes<sup>&reg;</sup> (CDK)** includes a single
+instance of the kube-api-loadbalancer.  For many use cases this is perfectly adequate,
+but in a production environment you should be keen to eliminate any single point of
+failure.
+
+The recommended way to provide a failover for the kube-api-loadbalancer is by using
+[keepalived][keepalived-home]. This is available as a **Juju** charm and can be deployed
+into your **CDK** model and configured as follows:
+
+1. Deploy the `keepalived` charm:
+    ```bash
+    juju deploy cs:~containers/keepalived-3
```

Comment:
version 2 is currently still the default in cs:, but 3 is required for bionic

Reply:
I've released rev 3 to stable so it's the default now.